src/bpftrace.h

```diff
@@ -51,10 +52,12 @@ inline DebugLevel operator++(DebugLevel& level, int)
 class BPFtrace
 {
 public:
-  BPFtrace() : ncpus_(get_possible_cpus().size()) { }
+  BPFtrace() : out_(std::cout),ncpus_(get_possible_cpus().size()) { }
+  BPFtrace(std::ostream& o) : out_(o),ncpus_(get_possible_cpus().size()) { }
```

Comment:
Could we just do this using a single constructor with a default value for `o`? I don't like duplicating code (`get_possible_cpus().size()`) if we can help it

Reply:
done